src/rpc/rawtransaction.cpp

```diff
@@ -302,6 +548,86 @@ UniValue gettxoutproof(const UniValue &params, bool fHelp)
     return strHex;
 }
 
+UniValue gettxoutproofs(const UniValue &params, bool fHelp)
+{
+    if (fHelp || params.size() != 2)
+        throw runtime_error(
+            "gettxoutproofs [\"txid\",...] ( blockhash )\n"
```

Comment:
if blockhash is optional, use [], since txid is not optional, use ()

Reply:
the reason this takes a JSON list is because arg is a direct copy of gettxoutproof's args which also takes a JSON list. block hash is no longer optional though